test/Microsoft.AspNet.Mvc.TagHelpers.Test/ScriptTagHelperTest.cs

```diff
@@ -291,5 +431,23 @@ private TagHelperLogger<ScriptTagHelper> CreateLogger()
         {
             return new TagHelperLogger<ScriptTagHelper>();
         }
+
+        private static IHostingEnvironment MakeHostingEnvironment(IFileProvider webRootFileProvider = null)
```

Comment:
Not seeing where you ever provide this with a file Provider, is the param needed?